README.md

```diff
@@ -1,27 +1,37 @@
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7108558.svg)](https://doi.org/10.5281/zenodo.7108558)
+[![PyPI](https://img.shields.io/pypi/v/glasflow)](https://pypi.org/project/glasflow/)
+[![Conda Version](https://img.shields.io/conda/vn/conda-forge/glasflow.svg)](https://anaconda.org/conda-forge/glasflow)
+
 # Glasflow
 
 glasflow is a Python library containing a collection of [Normalizing flows](https://arxiv.org/abs/1912.02762) using [PyTorch](https://pytorch.org). It builds upon [nflows](https://github.com/bayesiains/nflows).
 
 ## Installation
 
-To install from GitHub:
+glasflow is available to install via `pip`:
+
+```shell
+pip install glasflow
+```
+
+or via `conda`:
 
 ```shell
-$ pip install git+https://github.com/igr-ml/glasflow.git
+conda install glasflow -c conda-forge
 ```
 
 ## PyTorch
 
-By default the version of PyTorch will not necessarily match the drivers on your system, to install a different version with the correct CUDA support see the PyTorch homepage for instructions: https://pytorch.org/.
+By default the version of PyTorch installed by `pip` or `conda` will not necessarily match the drivers on your system, to install a different version with the correct CUDA support see the PyTorch homepage for instructions: https://pytorch.org/.
```

Comment:
Should we add here that you do not necessarily need CUDA to use nessai?

Reply:
I've added a note and tried to emphasise that it's not needed and in most cases doesn't provide any benefit.